_posts/en/newsletters/2020-04-15-newsletter.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Bitcoin Optech Newsletter #93'
+permalink: /en/newsletters/2020/04/15/
+name: 2020-04-15-newsletter
+slug: 2020-04-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a proposal for creating a unified
+multi-wallet backup that circumvents the inability to import BIP32
```

Comment:
Not worth mentioning (or too long to do so), but as it happened multiwallet API was a topic of discussion last week as well: https://github.com/bitcoin/bitcoin/pull/18453 (discussion toward the end about how to add a multiwallet API) and https://bitcoincore.reviews/18453.